src/App.js

```diff
@@ -30,42 +33,98 @@ const App = () => {
   // This starts state off as a 2D array of JS objects with
   // empty value and unique ids.
   const [squares, setSquares] = useState(generateSquares());
+  const [currentPlayer, setCurrentPlayer] = useState(PLAYER_1);
```

Comment:
For the most part, you seem to be in the no-semicolon camp. Most of the code with semicolons is in the boilerplate code, and most of yours lacks semicolons. Personally, I'm in the pro-semicolon camp. Strictly speaking semicolons in JS are NOT optional, and there are very real (though obscure) situations where lack of semicolons will result in errors.

For the most part, consistency is the most important thing, but be sure follow the conventions of any team that you join.